### ISSUE TITLE
Font misalignment

### ISSUE THREAD

**Issue by igor-borovkov** [WebPlatformForEmbedded/WPEWebKit#379]
For some fonts text is rendered out of place when trying to align it in the center of a div element. This issue is not reproducible on Chrome Mac, Safari Mac (but reproducible on Chrome Ubuntu, Firefox Ubuntu)

Example page:
http://test.borovkov.com/font/font2.html
1st character: default font (Dejavu Sans in my case)
2nd character: Xfinity Sans font (font-face, misaligned)
3rd and 4th:  some random downloadable fonts (alignment seems to be fine)

Each character is supposed to be located in the middle of a particular DIV element. Span element is used (blue colored) to highlight where a selection would be around the character. The issue is reproducible whether the span element is used or not. 

It seems that this issue might be related to a bug with a font file, there is a version of Xfinity Sans font file which was sort of modified so this is not reproducible, but since this bug is not reproducible with the original Xfinity Sans font on Mac OS I guess it's possible this can be fixed on WPE side. I have another scenario where this issue is reproducible for 0swald font (4th character), but not reproducible on the page above.

**Comment by magomez** [comment]
I'm pretty sure that this is a bug in the font file. I'm able to reproduce the issue with WebKit, Chrome and Firefox in my linux system, and all of them show the same behaviour, so it's not a webkit-only issue. My bet is that the font is somehow not working fine with FreeType, which is the cairo font backend used in linux, and that's why the problem is reproducible in every browser that uses them.
The problem is not happening on macOS because they don't use FreeTypeI guess. Maybe they have their own libraries, maybe a modified version of it (I'm not that familiar with macOS).
I found similar problems related to fonts before and in the end they were fixed by fixing the font.
